### PR TITLE
Also update product version in Staging1 TestDVD

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

**Comment by coolo** [comment]
that is going to break SLE hard IMO

**Comment by nilxam** [comment]
ah, indeed, I'll fix it for openSUSE only.

**Comment by nilxam** [comment]
fixed.

File: osclib/freeze_command.py [hunk 1]
```diff
@@ -140,8 +140,14 @@ def perform(self, prj):
             time.sleep(1)
         self.build_switch_bootstrap_copy('disable')
 
+        # Update the version information found in the Test-DVD package, to match openSUSE-release
+        if self.api.item_exists(prj, "openSUSE-release"):
+            version = self.api.package_version(prj, 'openSUSE-release')
+            for arch in ['x86_64', 'ppc64le']:
```
Comment (lnussel) [inline osclib/freeze_command.py]: this should use self.cstaging_dvd_archs

Comment (nilxam) [inline osclib/freeze_command.py]: what is it?

Comment (lnussel) [inline osclib/freeze_command.py]: for arch in self.api.cstaging_dvd_archs:

**Pull request merged**